media-libs/openh264/openh264-2.3.1-r1.ebuild

```diff
@@ -54,6 +54,7 @@ emakecmd() {
 		INCLUDES_DIR="${EPREFIX}/usr/include/${PN}" \
 		HAVE_AVX2=$(usex cpu_flags_x86_avx2 Yes No) \
 		ARCH="$(tc-arch)" \
+		ENABLEPIC="Yes" \
```

Comment:
Needs a new revision (it changes installed files). I suppose you could make it conditional on USE=pic instead but I don't really like that USE flag and may try to kill it off at some point. So, looks fine other than new revision needed.